about/licenses/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: LLNL Software Licensing
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+*This page contains information meant for LLNL Employees with instructions for
+how to mark their source code projects. It is not intended to serve as legal
+advice*
+
+### Required Content
+
+All LLNL software must contain the following markings:
+
+#### 1. README file
+
+All projects shall have a `README.md` file at the root of the repository,
+including at least the following information:
+
+- "Getting Started" instructions
+- "Releasability" section, containing the `LLNL-CODE-XXXXXX` number
```

Comment:
Is this the only place for the CODE number? I think lots of projects right now use a RELEASE file.

Reply:
Hmm, do you have any examples? I think this is going to end up being the only one we need. There is some discussion about having a top level metadata file that would contain the number as well, but that's still a work in progress.

Reply:
- https://github.com/LLNL/RAJA/blob/develop/RELEASE
- https://github.com/LLNL/Umpire/blob/develop/RELEASE

Reply:
Maybe "lots" was wrong.... 😄

Reply:
Many of the projects I'm familiar with have a release section in the readme.

Reply:
Perhaps... ;)

I think having a `RELEASE` file like that looks good actually, though I would argue for it being in Markdown format (thus `RELEASE.md`). That way the information is in a known place, rather than cluttering up the `README`.

Reply:
Probably if we do have it as a separate file we would need something in the README to point to the: RELEASE, DISCLAIMER, and LICENSE files anyways.

Reply:
I'm kind of opposed to proliferating top-level repository files.  RELEASE is yet another thing to look at when you do an `ls`, and it's like one line.  Can't we just advise people to put this in the README?

Reply:
LESS FILES.

Reply:
if you go the opposite of less files....

I've always liked a metadata.yaml (or something similar) that's more machine-parsable with pertinent project information. This would make it easy to pull stats and details in mass at a later time by iterating the organization and parsing these files, vs doing more full text searches and pattern matching elsewhere. Just a thought

Reply:
If there were a recommended standard for something like this it would be great but what they're proposing here is  RELEASE just for our LLNL release designator.  Do other projects have `metadata.yaml`?

Reply:
different orgs do it different ways, so there is no standard. Netflix was pushing OSSMETADATA for a while, but was very limited in the information that it supported. 

The Finos foundation uses a similar file which relates to a buch of info, you can see a sample at : https://github.com/finos/finos.github.io/blob/master/activities.json